source/runtime_gui.cpp

```diff
@@ -2280,14 +2285,16 @@ void reshade::runtime::draw_gui_settings()
 		modified |= ImGui::Checkbox(_("Save before and after images"), &_screenshot_save_before);
 #endif
 		modified |= ImGui::Checkbox(_("Save separate image with the overlay visible"), &_screenshot_save_gui);
-
+		modified |= ImGui::Checkbox(_("Split into folders by app name"),&_screenshot_path_split_appname);
+	if (ImGui::IsItemHovered(ImGuiHoveredFlags_ForTooltip))
+		{
+        ImGui::SetItemTooltip(_("Split save paths into folders by App Name.\nUseful if you have multiple games sharing the same screenshot directory"));
+		}
```

Comment:
With the above proposal this option would go away and instead would just add the same tooltip on the "Screenshot name" widget to the "Screenshot path" widget too.